misc/launchctl/io.ipfs.ipfs-daemon.plist

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--
+  A bare-bones launchctl agent file for ipfs. To have launchctl automatically run the ipfs daemon for you, do the following:
+
+      mkdir -p ~/Library/LaunchAgents
+      cp misc/launchctl/io.ipfs.ipfs-daemon.plist ~/Library/LaunchAgents/
+      launchctl load ~/Library/LaunchAgents/io.ipfs.ipfs-daemon.plist
+
+  Note that the `ipfs` binary must be on the *system* PATH for this to work. Adding a symlink in /usr/bin works well enough for me.
+-->
+<plist version="1.0">
+  <dict>
+    <key>KeepAlive</key>
+    <true/>
+    <key>Label</key>
+    <string>io.ipfs.ipfs-daemon</string>
+    <key>ProgramArguments</key>
+    <array>
+      <string>ipfs</string>
+      <string>daemon</string>
+    </array>
+    <key>RunAtLoad</key>
+    <true/>
+    <key>WorkingDirectory</key>
+    <string>/usr/local/var</string>
+    <key>StandardErrorPath</key>
+    <string>/usr/local/var/log/ipfs.err</string>
+    <key>StandardOutPath</key>
+    <string>/usr/local/var/log/ipfs.log</string>
```

Comment:
if the working directory is `/usr/local/var`, then maybe we should put the repo there instead of home directory? (Can do so setting `IPFS_PATH=<path>`). not sure. OTOH, we could make these go into the default `$IPFS_PATH` (`~/.ipfs`)